src/test/java/Elements/VariousElements.java

```diff
@@ -0,0 +1,31 @@
+package Elements;
+
+import io.cucumber.datatable.DataTable;
+import org.openqa.selenium.By;
+
+public class VariousElements {
```

Comment:
create a class for each page and put elements per page. also create a class for the menu and one class for the footer

Reply:
Will do so...